File: ApplicationLogic/src/main/java/dk/via/sep3/group1/applicationlogic/model/CreditCard.java
package dk.via.sep3.group1.applicationlogic.model;

import com.fasterxml.jackson.annotation.JsonProperty;


public class CreditCard {

    @JsonProperty("creditCardNumber")
    private String creditCardNumber;
    @JsonProperty("firstName")
    private String firstName;
    @JsonProperty("lastName")
    private String lastName;
    @JsonProperty("expirationMonth")
    private String expirationMonth;
    @JsonProperty("expirationYear")
    private String expirationYear;
    @JsonProperty("securityCode")
    private int securityCode;
    @JsonProperty("amountOfMoney")
    private float amountOfMoney = 900000;
    @JsonProperty("account")
    private Account account;

    public CreditCard() {
    }

    public String getCreditCardNumber() {
        return creditCardNumber;
    }

    public void setCreditCardNumber(String creditCardNumber) {
        this.creditCardNumber = creditCardNumber;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getExpirationMonth() {
        return expirationMonth;
    }

    public void setExpirationMonth(String expirationMonth) {
        this.expirationMonth = expirationMonth;
    }

    public String getExpirationYear() {
        return expirationYear;
    }

    public void setExpirationYear(String expirationYear) {
        this.expirationYear = expirationYear;
    }

    public int getSecurityCode() {
        return securityCode;
    }

    public void setSecurityCode(int securityCode) {
        this.securityCode = securityCode;
    }

    public float getAmountOfMoney() {
        return amountOfMoney;
    }

    public void setAmountOfMoney(int amountOfMoney) {
        this.amountOfMoney = amountOfMoney;
    }

    public Account getAccount() {
        return account;
    }

    public void setAccount(Account account) {
        this.account = account;
    }
}

File: PresentationLayer/PresentationLayer/Models/CreditCard.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PresentationLayer.Models
{
    public class CreditCard
    {
        public string CreditCardNumber { get; set; }
        
        public string FirstName { get; set; }
        
        public string LastName { get; set; }
        
        public string ExpirationMonth { get; set; }

        public string ExpirationYear { get; set; }

        public int SecurityCode { get; set; }

     
        public float AmountOfMoney { get; set; }
        
        public Account Account { get; set; }

        public CreditCard(string creditCardNumber, string firstName, string lastName, string expirationMonth, string expirationYear, int securityCode, float amountOfMoney, Account account)
        {
            CreditCardNumber = creditCardNumber;
            FirstName = firstName;
            LastName = lastName;
            ExpirationMonth = expirationMonth;
            ExpirationYear = expirationYear;
            SecurityCode = securityCode;
            AmountOfMoney = amountOfMoney;
            Account = account;
        }
    }
}